datasets/Wijnen_2024/compose.py

```diff
@@ -0,0 +1,37 @@
+from asreview import ASReviewData
+import pandas as pd
+from pathlib import Path
+import numpy as np
+
+key = "Wijnen_2024"
+
+# Data comes from: https://zenodo.org/records/13957522
+search = ASReviewData.from_file("https://zenodo.org/records/13957522/files/Initial%20search%20(outreach%20included).ris").df
+search["label_included"] = 0
+search["label_abstract_included"] = 0
+
+ti_ab_exclude = ASReviewData.from_file("https://zenodo.org/records/13957522/files/Excluded%20first%20round.ris").df
+ti_ab = search[~search[['primary_title']].apply(lambda x: np.in1d(x,ti_ab_exclude).all(),axis=1)].reset_index(drop=True)
```

Comment:
Is this an anti-join? I think it's easier to read with pandas `~isin`.

Reply:
Yes, indeed looks better now.